enhancements/baremetal/an-slo-for-baremetal.md

```diff
@@ -237,8 +238,18 @@ The CBO will:
   currently creates. In other words, the initial purpose of the CBO
   will be to reconcile the `Provisioning` singleton resource.
 
+#### Operator framework
+
+CBO will be built using [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder). It has been decided to adopt it
+since it is compatible with the `operator-sdk`.
```

Comment:
```suggestion
since it is compatible with the `operator-sdk` and the `operator-sdk` does not
provide any additional features needed by the new operator.
```